lms/static/scripts/frontend_apps/components/GradingToolbar.tsx

```diff
@@ -71,8 +69,7 @@ export default function LMSGrader({
    * Makes an RPC call to the sidebar to change to the focused user.
    */
   const changeFocusedUser = useCallback(
-    /** @param {StudentInfo|null} user - The user to focus on in the sidebar */
-    async user => {
+    async (user: StudentInfo | null) => {
```

Comment:
I deleted a couple of comments which, IMO, did not add value. eg. Here there is a comment above which says the same thing.